### PR TITLE
[4.0] Fixing display Notice in Associations tab when item language is set to All

### PR DESCRIPTION
**Pull request by infograf768**
Pull Request for Issue https://github.com/joomla/joomla-cms/issues/27017#issuecomment-551591426

### Summary of Changes
In https://github.com/joomla/joomla-cms/commit/3b93c7c86e404804c3ac1a3183c39eb285e26d6f#diff-2aa7782ed45d0b90a0cc2b6fbd85fd19 was introduced a variable preventing to display the message in Associations tab when the item language is set to All

### Testing Instructions
Install a multilingual site
Create and save an article, a category, a contact, a newsfeed, a menu item, with language set to ALL.
Edit these items and display the Associations tab.

### Before patch

Screen Shot 2019-11-08 at 12 03 56

### After patch

Screen Shot 2019-11-08 at 12 01 34

### CODE REVIEW

File: administrator/components/com_contact/tmpl/contact/edit.php [hunk 1]
```diff
@@ -122,14 +121,12 @@
...
-				<legend><?php echo Text::_('JGLOBAL_FIELDSET_ASSOCIATIONS'); ?></legend>
-				<div>
-				<?php echo LayoutHelper::render('joomla.edit.associations', $this); ?>>
-				</div>
-				</fieldset>
-			<?php endif; ?>
+			<fieldset id="fieldset-associations" class="options-grid-form options-grid-form-full">
+			<legend><?php echo Text::_('JGLOBAL_FIELDSET_ASSOCIATIONS'); ?></legend>
+			<div>
+			<?php echo LayoutHelper::render('joomla.edit.associations', $this); ?>>
```
Comment (Quy) [inline administrator/components/com_contact/tmpl/contact/edit.php]:
```suggestion
			<?php echo LayoutHelper::render('joomla.edit.associations', $this); ?>
```

Comment (infograf768) [reply administrator/components/com_contact/tmpl/contact/edit.php]: done. tks.

**Comment by Quy** [comment]
Probably beyond the scope of this PR that the message be displayed in the box below it or remove the box.

**Comment by richard67** [comment]
@Quy For me it is ok: The alert is shown where alerts shall be shown, and the associations box is there where it always is but empty. I think this is consistent.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 43569af3ed2081e6ca0dd49b9e65c02b0ba0bc2c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27023">issues.joomla.org/tracker/joomla-cms/27023</a>.</sub>

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 43569af3ed2081e6ca0dd49b9e65c02b0ba0bc2c<br /><br />In addition to testing instructions, tested that the notice is not shown for an item which does not have associations but is set to a specific language (i.e. not "All").<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27023">issues.joomla.org/tracker/joomla-cms/27023</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27023">issues.joomla.org/tracker/joomla-cms/27023</a>.</sub>

**Comment by HLeithner** [comment]
Thanks

**Pull request merged**